site/content/getting-started.md

```diff
@@ -18,6 +18,12 @@ As noted above, Hono consists of multiple service components that together compr
 You will need at least Docker version 1.13.0 and Docker Compose version 1.10.0 in order to run the example in this guide. By the time of this writing, the latest released versions of Docker and Docker Compose were 1.13.1 and 1.11.1 respectively.
 {{% /warning %}}
 
+Please do not forget to set your docker to swarm mode first, e.g. like
+
+    docker swarm init --advertise-addr <your-swarm-advertiser>
```

Comment:
I don't think that you need to set the advertise address (this is only relevant if you plan to add more nodes to the swarm)